README.md

```diff
@@ -46,31 +48,35 @@ repositories {
 
 ## Usage
 
-Create a `DropInRequest` and use the `Intent` to start Drop-In:
+Create a `DropInRequest` to start the Drop-in UI with specified options:
 
-```java
-DropInRequest dropInRequest = new DropInRequest()
-    .clientToken(mClientToken);
-startActivityForResult(dropInRequest.getIntent(context), DROP_IN_REQUEST);
+```kotlin
```

Comment:
I opted to switch to Kotlin snippets since we already have Java example code in the demo app - can put it back to Java if we don't like that change though

Reply:
Are Kotlin and Java both common enough that we'd want example code for both?

Reply:
They are both common, I've been seeing Kotlin more in merchant inquiries/issues. We have snippets for both on the docs site, so could easily paste the Java snippets here too, it just might make the README a bit long. I don't feel strongly either way.

Reply:
We usually go for a Kotlin-first approach in the migration guides but we provide snippets for both Java and Kotlin on the BT docs site.

Reply:
I'm fine to use Kotlin as the default, but maybe a happy middle ground would be to just drop a quick link to the Java snippets to make it more discoverable?

Reply:
That's a good call. The developer docs for v6 haven't been released yet, so I'll set a reminder for myself to add that link in once those docs are released.